plugin/src/test/java/org/jenkinsci/plugins/structs/describable/DescribableModelTest.java

```diff
@@ -53,12 +60,16 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import static org.bouncycastle.asn1.x500.style.RFC4519Style.dc;
 import static org.jenkinsci.plugins.structs.describable.DescribableModel.*;
+import static org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable.ANONYMOUS_KEY;
 import static org.junit.Assert.*;
 import org.junit.Ignore;
 
 @SuppressWarnings("unchecked") // generic array construction
 public class DescribableModelTest {
+    @ClassRule
+    public static JenkinsRule rule = new JenkinsRule();
```

Comment:
Would not be necessary if you fixed `SymbolLookup` to fall back to `Thread.contextClassLoader` as `DescribableModel` already did.